### ISSUE TITLE
Everyone update the slide decks for your sessions

### ISSUE THREAD

**Issue by daniellecrobinson** [daniellecrobinson/mini-wow-pdx#11]
https://daniellecrobinson.github.io/mini-wow-pdx/

Check the side deck, make a copy, edit it, update index.html, or email Danielle

README @teonbrooks 
Contributors and code of conduct @rchampieux 
Roadmapping @daniellecrobinson
Open Canvas @dasaderi

**Comment by daniellecrobinson** [comment]
Ahem ... just lemme know when there are updated y'all <3

**Comment by dasaderi** [comment]
Sorry, I will work on it tonight.

**Comment by rchampieux** [comment]
Thanks for the reminder!  I will be working on all stuffs WOW tomorrow!

**Comment by dasaderi** [comment]
I updated the link to my open canvas presentation. Let me know if it doesn't work and/or you want me to change something. Thank you.

**Comment by daniellecrobinson** [comment]
It looks great! @dasaderi

**Comment by daniellecrobinson** [comment]
To @teonbrooks @rchampieux @stephwright I've made copies of the WOW presentations and am updating the links now. You shoudl then be able to just update directly on to the presentation

README: https://docs.google.com/presentation/d/11-eY9gHF2DP6uSqKYfyI7ht5nexOoiRWWm7C-zEVv_E/edit?usp=sharing
Contributor and CoC: https://docs.google.com/presentation/d/1snF8yL5QZr4UD203Zhm_YxplURdFm56ljeLPdNm5g_A/edit#slide=id.g1088c5b110_0_183

**Comment by stephwright** [comment]
@teonbrooks I took out the Aurelia slide and updated the slides w/ the upgoer5 link.  Those two slides and their placement are good for me to do the piece on jargon busting.  How do they look to you?

**Comment by daniellecrobinson** [comment]
I AM DONE WITH ROADMAPPING AND ALMOST DONE WITH GITHUB 🙌

**Comment by rchampieux** [comment]
Mine will be done tomorrow afternoon.

**Comment by rchampieux** [comment]
@daniellecrobinson I don't think we finalized the what exercise I'd focus on (as we don't have time for both right).  Okay with everyone if I do contributor guidelines?
